### DISCUSSION TITLE
Animate object along a svg path.

### DISCUSSION THREAD

**Discussion by morganp2** [pmndrs/react-spring#1727] [Support] [answered]
Hi All,

I have only just started using react-spring so I don't fully understand what is possible. 

I have gone through the examples, issues, discussions and a number of codesandboxes but can't find anything related to animating objects along a SVG path.

I came across this example using framer-motion:
https://codesandbox.io/s/framer-motion-motion-along-a-path-41i3v

Is this functionality available in react-spring?

Thanks,

**Comment by joshuaellis** [comment, marked as answer]
You sure can, apologise for the messy sandbox (i'm at work)

https://codesandbox.io/s/react-spring-motion-along-a-path-yv8mn?file=/src/styles.css

**Reply by morganp2** [reply]
Hi Josh,
That is cool. Thanks!!
